### PR TITLE
Merge altmetrics docs

### PR DESCRIPTION
**Pull request by yoannspace**
The altmetrics docs (incl widget) can be merged in the metrics_portal

### CODE REVIEW

File: src/pages/Docs/markdown/widget.en-GB.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<!-- Customise Widget parameters -->
+<script>
+    let widget_params = {
+        "uri": "info:doi:10.5334/bbc",
+    };
+</script>
+
+<!-- load js for the Metrics Widget app-->
+<script src="https://storage.googleapis.com/operas/metrics-widget/hirmeos-metrics-0.1.1.min.js"></script>
```
Comment (rowan08) [inline src/pages/Docs/markdown/widget.en-GB.md]: This should also be `0.2.0`

File: src/pages/Docs/markdown/altmetrics.en-GB.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Almetrics
```
Comment (rowan08) [inline src/pages/Docs/markdown/altmetrics.en-GB.md]: Typo: "Altmetrics"

**Comment by rowan08** [comment]
Two minor changes, but otherwise, looks good :+1:

File: src/pages/Docs/markdown/widget.en-GB.md (hunk 1 above)
Comment (yoannspace) [reply src/pages/Docs/markdown/widget.en-GB.md]: Fixed

File: src/pages/Docs/markdown/altmetrics.en-GB.md (hunk 2 above)
Comment (yoannspace) [reply src/pages/Docs/markdown/altmetrics.en-GB.md]: Fixed

**Pull request merged**